data/hai outfits.txt

```diff
@@ -0,0 +1,396 @@
+		"stream"
+	description "Centuries ago, the Hai fought with more rudimentary versions of the Tracker, and this was the primary defence against them.  In spite of the dated design, it is still occasionally effective against their latest Missile weapons."
+
+outfit "Chameleon Anti-Missile Turret"
+	category "Turrets"
+	cost 150000
+	thumbnail "outfit/heavy anti-missile hai"
+	"mass" 32
+	"outfit space" -28
+	"weapon capacity" -28
+	"turret mounts" -1
+	weapon
+		"hit effect" "large anti-missile"
+		"anti-missile" 10
+		"velocity" 200
+		"lifetime" 1
+		"reload" 5
+		"firing energy" 11
+		"firing heat" 5.2
+	description "The Chameleon Anti-Missile turret has a longer range, higher power, and shorter reload time than the Bullfrog. The system is less vulnerable to large clusters of incoming missiles, especially when many turrets are installed."
+
+outfit "Hai Williwaw Cooling"
+	category "Systems"
+	cost 15000
+	thumbnail "outfit/cooling ducts hai"
+	"mass" 7
+	"outfit space" -7
+	"cooling" 8.3
+	description "While Hai technology is fairly heat efficient, occasionally they reach the point where their ship sporadically overheats and shuts down. When that happens, they install these."
+
+outfit "Hai Chasm Batteries"
+	category "Power"
+	cost 8000
+	thumbnail "outfit/tiny battery hai"
+	"mass" 5
+	"outfit space" -5
+	"energy capacity" 550
+	description "Power storage technology is thousands of years old.  This is the smallest Hai battery pack."
+
+outfit "Hai Fissure Batteries"
+	category "Power"
+	cost 25000
+	thumbnail "outfit/small battery hai"
+	"mass" 15
+	"outfit space" -15
+	"energy capacity" 1800
+	description "This battery pack has more than three times the energy capacity of the the Hai Chasm Batteries, allowing their ships to engage in combat for longer periods of time before running out of energy."
+
+outfit "Hai Gorge Batteries"
+	category "Power"
+	cost 52000
+	thumbnail "outfit/medium battery hai"
+	"mass" 30
+	"outfit space" -30
+	"energy capacity" 3750
+	description "This medium-sized battery pack is ideal for most of the Hai power needs, allowing for medium length combat engagements, but could also be used keep a ship's heat footprint low with a small power generator and oversized engines."
+
+outfit "Hai Ravine Batteries"
+	category "Power"
+	cost 120000
+	thumbnail "outfit/large battery hai"
+	"mass" 60
+	"outfit space" -60
+	"energy capacity" 8250
+	description "This large battery pack boasts higher energy density than smaller models, and helps keep ships fighting in spite of sustaining several Ion Cannon impacts."
+
+outfit "Hai Valley Batteries"
+	category "Power"
+	cost 300000
+	thumbnail "outfit/huge battery hai"
+	"mass" 120
+	"outfit space" -120
+	"energy capacity" 20000
+	description "While most Hai ships do not employ their largest battery model, a few very powerful warships are occasionally equipped with one, keeping their heat footprint low, and in a fight against Ion Cannon equipped opponents through to victory."
```

Comment:
and in a fight against Ion Cannon equipped opponents through to victory - this doesn't really make sense if you think about it.